src/modules/username.rs

```diff
@@ -129,6 +136,18 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn ssh_connection_tty() -> io::Result<()> {
+        let actual = ModuleRenderer::new("username")
+            .env("USER", "astronaut")
+            .env("SSH_TTY", "/dev/pts/0")
+            .collect();
+        let expected = Some(format!("{} in ", Color::Yellow.bold().paint("astronaut")));
+
+        assert_eq!(expected, actual);
+        Ok(())
+    }
+
```

Comment:
Can you add a test for the `SSH_CLIENT` env var?

Reply:
Test added. I also included some documentation on the behavior, and a workaround on servers that do not set these environment variables correctly.